modules/button/react/lib/IconButton.tsx

```diff
@@ -192,10 +192,15 @@ export default class IconButton extends React.Component<IconButtonProps> {
 
   public render() {
     // onToggleChange will generate a warning if spread over a <button>
-    const {onToggleChange, ...iconButtonConProps} = this.props;
+    const {buttonRef, buttonSize, onToggleChange, ...iconButtonConProps} = this.props;
```

Comment:
Can you pull out `toggled` as well and change `iconButtonConProps` to `elemProps` a la https://github.com/Workday/canvas-kit/blob/master/API_PATTERN_GUIDELINES.md#prop-spread-behavior

Reply:
yep!